inputs/orchestrator_inner.json

```diff
@@ -0,0 +1,37 @@
+{
+  "client_version": "{{VERSION}}",
+  "prebuild_plugins": [
+    {
+      "name": "pull_base_image",
+      "args": {
+        "parent_registry": "{{REGISTRY_URI}}",
+        "parent_registry_insecure": true
+      }
+    },
+    {
+      "name": "bump_release",
+      "args": {
+        "hub": "{{KOJI_HUB}}"
+      }
+    },
+    {
+      "name": "reactor_config",
+      "args": {
+        "config_path": "{{CONFIG_PATH}}"
+      }
+    }
+  ],
+  "buildstep_plugins": [
+    {
+      "name": "orchestrate_build",
+      "args": {
+        "platforms": "{{PLATFORMS}}",
+        "build_kwargs": "{{BUILD_KWARGS}}",
+        "osbs_client_config": "{{OSBS_CLIENT_CONFIG}}"
+      }
+    }
+  ],
+  "postbuild_plugins": [],
+  "prepublish_plugins": [],
+  "exit_plugins": []
```

Comment:
If we pull the parent image we'll also need to run `remove_built_image` on exit to remove it.